Python/bytecodes.c

```diff
@@ -2400,6 +2399,7 @@ dummy_func(
             assert(tstate->previous_executor == NULL);
             tstate->previous_executor = Py_None;
             Py_INCREF(executor);
+            CHECK_EVAL_BREAKER();
```

Comment:
This concerns me a bit. `CHECK_EVAL_BREAKER()` contains a `goto error` and I'm worried that that would leak the executor.

Reply:
Thanks for fixing this!